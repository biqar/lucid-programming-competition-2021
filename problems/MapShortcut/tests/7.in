6
Right 30 10
Right 120 10
Right 60 10
Right 60 10
Right 60 10
Right 60 10
