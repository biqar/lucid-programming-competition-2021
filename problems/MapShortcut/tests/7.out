Left 30 10
